### ISSUE TITLE
Cascade-Forward Neural Network Based on Resilient Backpropagation for Simultaneous Parameters and State Space Estimations of Brushed DC Machines. (arXiv:2104.04348v1 [eess.SP])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#5768]
Cascade-Forward Neural Network Based on Resilient Backpropagation for Simultaneous Parameters and State Space Estimations of Brushed DC Machines. (arXiv:2104.04348v1 [eess.SP])<br>
https://ift.tt/3mFT5MA<br>
<p>A sensorless speed, average temperature and resistance estimation technique based on Neural Network (NN) for brushed DC machines is proposed in this paper. The literature on parameters and state spaces estimations of the Brushed DC machines, shows a variety of approaches. However, these observers are sensitive to a noise, on the model accuracy also are difficult to stabilize and to converge. Furthermore, the majority of earlier works, estimate either the speed or the temperature or the winding resistance. According to the literatures, the Resilient backpropagation (RBP) as is the known as the faster BP algorithm, Cascade-Forward Neural Network (CFNN), is known as the among accelerated learning backpropagation algorithms, that's why where it is found in several researches, also in several applications in these few years. The main objective of this paper is to introduce an intelligent sensor based on resilient BP to estimate simultaneously the speed, armature temperature and resistance of brushed DC machines only from the measured current and voltage. A comparison between the obtained results and the results of traditional estimator has been made to prove the ability of the proposed method. This method can be embedded in thermal monitoring systems, in high performance motor drives.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
